### DISCUSSION TITLE
Feature Request: custom URL “slug”

### DISCUSSION THREAD

**Discussion by gsn1511** [AndrewPaglusch/FlashPaper#181] [Ideas]
Instead of the randomly generated string at example.com/k?[SLUG], make a way to create a custom "slug".

**Comment by AndrewPaglusch** [comment]
Hey @gsn1511 , thanks for the feature request.

The random string in the URL is the secret's ID and decryption key, so replacing it with something user-provided would break the security of FlashPaper.

Thanks!

**Reply by gsn1511** [reply]
Thanks for the fast reply. The answer was very obvious apparently, sorry for that. Keep up the good work!

**Reply by AndrewPaglusch** [reply]
You're welcome!
